src/Kunstmaan/AdminListBundle/Controller/AdminListController.php

```diff
@@ -216,6 +226,51 @@ protected function doEditAction(AbstractAdminListConfigurator $configurator, $en
             throw new AccessDeniedHttpException('You do not have sufficient rights to access this page.');
         }
 
+        if ($helper instanceof LockableEntityInterface) {
```

Comment:
Can you extract this whole chunk into a separate service (or multiple services)?